### PR TITLE
Remove mentions of Python 2

### PR DESCRIPTION
**Pull request by gregvanl**
Fixes https://github.com/microsoft/vscode-docs/issues/5954

I think the only explicit mention was in the Black formatter section but cleaned up a couple Python versioning examples as well.

### CODE REVIEW

**Comment by gregvanl** [comment]
Leaving a comment for testing

File: docs/python/environments.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ If an interpreter hasn't been specified, then the Python extension automatically
 
   1. Virtual environments located directly under the workspace folder.
   2. Virtual environments related to the workspace but stored globally. For example, [Pipenv](https://pypi.org/project/pipenv/) or [Poetry](https://python-poetry.org/) environments that are located outside of the workspace folder.
-  3. Globally installed interpreters. For example, the ones found in `/usr/local/bin`, `C:\\python27`, `C:\\python38`, etc.
+  3. Globally installed interpreters. For example, the ones found in `/usr/local/bin`, `C:\\python38`, etc.
```
Comment (gregvanl) [inline docs/python/environments.md:55]: Leaving an inline comment for testing

File: learn/develop-cloud/overview.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Remote development has benefits such as:
 
 ### Example scenario
 
-As an example scenario, you could have different dev environments for Python 2.7 and 3.7. Rather than having to worry about managing different versions of Python on your computer, you can connect to separate environments with the appropriate versions of Python already set up for you.
+As an example scenario, you could have different dev environments for different versions of Python. Rather than having to worry about managing different versions of Python on your computer, you can connect to separate environments with the appropriate versions of Python already set up for you.
```
Comment (gregvanl) [inline learn/develop-cloud/overview.md:26]: Another comment for testing

File: docs/python/editing.md [hunk 3]
```diff
@@ -142,10 +142,6 @@ The following settings apply to the individual formatters. The Python extension
 | black (see note) | pip install black | blackArgs | blackPath |
 | yapf | pip install yapf | yapfArgs | yapfPath |
 
-> **Note**: By default, the Black formatter can't be installed when a Python 2 environment is active. Attempting to do so may display the message "Formatter black is not installed. Install?". If you try to install Black in response, another message appears saying "Could not find a version that satisfies the requirement black' No matching distribution found for black."
->
-> To work around this issue and use the Black formatter with Python 2, first install Black in a Python 3 environment. Then set the `python.formatting.blackPath` setting to that install location.
-
 When using custom arguments, each top-level element of an argument string that's separated by space on the command line must be a separate item in the args list. For example:
```
Comment (gregvanl) [inline docs/python/editing.md:145]: Leaving an comment below a deletion

File: learn/develop-cloud/overview.md (hunk 2 above)
Comment (gregvanl) [inline learn/develop-cloud/overview.md:26]: Leaving a comment on a two sentence paragraph

**Review by karthiknadig — APPROVED** [review]

File: docs/python/editing.md [hunk 4]
```diff
@@ -142,10 +142,6 @@ The following settings apply to the individual formatters. The Python extension
 | black (see note) | pip install black | blackArgs | blackPath |
 | yapf | pip install yapf | yapfArgs | yapfPath |
```
Comment (gregvanl) [inline docs/python/editing.md:144]: leaving an inline comment on a deletion

File: docs/python/editing.md [hunk 5]
```diff
@@ -142,10 +142,6 @@ The following settings apply to the individual formatters. The Python extension
...
 
-> **Note**: By default, the Black formatter can't be installed when a Python 2 environment is active. Attempting to do so may display the message "Formatter black is not installed. Install?". If you try to install Black in response, another message appears saying "Could not find a version that satisfies the requirement black' No matching distribution found for black."
->
-> To work around this issue and use the Black formatter with Python 2, first install Black in a Python 3 environment. Then set the `python.formatting.blackPath` setting to that install location.
-
 When using custom arguments, each top-level element of an argument string that's separated by space on the command line must be a separate item in the args list. For example:
```
Comment (gregvanl) [inline docs/python/editing.md:146]: A comment below a deletion

File: docs/python/editing.md (hunk 4 above)
Comment (gregvanl) [inline docs/python/editing.md:144]: Leaving a comment above a deletion

File: docs/python/editing.md [hunk 6]
```diff
@@ -142,10 +142,6 @@ The following settings apply to the individual formatters. The Python extension
 | black (see note) | pip install black | blackArgs | blackPath |
```
Comment (gregvanl) [inline docs/python/editing.md:142]: Leaving a comment near a deletion

**Pull request merged**